build_projects/dotnet-host-build/TestTargets.cs

```diff
@@ -105,7 +105,8 @@ private static List<string> RunDotnetTestOnTestProjects(BuildTargetContext c, Do
             bool fIsCrossArch = false;
             if (!String.IsNullOrEmpty(rid))
             {
-                if ((String.Compare(rid, "win8-arm", true) == 0) || (String.Compare(rid, "win10-arm64", true) == 0))
+                if ((String.Compare(rid, "win8-arm", true) == 0) || (String.Compare(rid, "win10-arm64", true) == 0)
```

Comment:
Since this will likely expand even more in future, can you please refactor it into a helper function?